webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpHeatMaps.tsx

```diff
@@ -240,22 +240,22 @@ const HpHeatMaps: React.FC<Props> = ({
                   minItemWidth={resize.width > 320 ? 35 : 27}
```

Comment:
I'm noticing something odd, which might be expected (and me ignorant). When clicking on some charts to zoom, the clicked chart vs the chart in the popup some time are not matching? (in the video attached, the 1st match, the 2nd and the 3rd doesn't, ...)


https://user-images.githubusercontent.com/5413702/114828524-4a462e00-9dca-11eb-9915-d4f9c28f5683.mov

Reply:
oops, missed that interaction. Thanks for catching it!